crates/node/src/cli/mod.rs

```diff
@@ -76,6 +97,21 @@ impl RunCmd {
         self.name.as_ref().expect("Name was just set")
     }
 
+    pub fn get_config(&self) -> anyhow::Result<Arc<ChainConfig>> {
+        let mut chain_config: ChainConfig = match &self.preset {
+            Some(preset_name) => ChainConfig::from_preset(preset_name.as_str())?,
+            None => {
+                ChainConfig::from_yaml(&self.chain_config_path.clone().expect("Failed to retrieve chain config path"))?
```

Comment:
Maybe a more explicit error for when a chain config has not been specified at all and no preset name was given?

Reply:
if neither a yaml or preset is given, it throw an error due to #[clap(group(ArgGroup::new("chain_config").args(["chain_config_path", "preset"]).required(true)))]